src/index.js

```diff
@@ -38,3 +37,19 @@ export const renderReactWithAphrodite = (name, component) => hypernova({
     return component;
   },
 });
+
+export const renderReactWithAphroditeStatic = (name, component) => hypernova({
+  server() {
+    return (props) => {
+      const { html, css } = StyleSheetServer.renderStatic(() => (
+        ReactDOMServer.renderToStaticMarkup(React.createElement(component, props))
+      ));
+
+      const style = `<style data-aphrodite>${css.content}</style>`;
```

Comment:
This mimics what's done in the non-static renderer. I can change it to `data-aphrodite="data-aphrodite"` for both, though, if you'd like that to happen in this PR.